CMakeLists.txt

```diff
@@ -5,7 +5,6 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/)
 include(DefaultCMakeBuildType)
 include(DownloadAndExtractLLVM)
 
-set(CLANG_VERSION 6.0.0 CACHE STRING "Downloaded Clang version (6.0.0)")
```

Comment:
Keep `CLANG_VERSION`? We can remove 4.0.0 5.0.1 compatibility code but keep it in hope of consistent tarball names in future releases... This gives users some flexibility of choices when some versions do not work

Reply:
I'll add it back. Although that's going to require a lot of if tests to set the names correctly for each version since they change all the time.